sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_client_async.py

```diff
@@ -240,8 +240,8 @@ def get_queue_sender(self, queue_name: str, **kwargs: Any) -> ServiceBusSender:
 
         :param str queue_name: The path of specific Service Bus Queue the client connects to.
```

Comment:
I'll save you from repetitive suggestions here... but same formatting nits as the sync versions.